### ISSUE TITLE
Serial editor has white background in High contrast

### ISSUE THREAD

**Issue by microsoftsam** [Microsoft/pxt#3573]
This is what the serial editor looks like in High contrast mode: 
<img width="1230" alt="screen shot 2017-12-05 at 4 03 21 pm" src="https://user-images.githubusercontent.com/16690124/33637455-eb56c622-d9d5-11e7-85b0-1b0ad9cddb0b.png">

It should have a dark background like the rest of the editor, and all colors should only use black white, highlight (teal blue) or green (for disabled) in accordance with the High contrast theme.
